R/drive_mime_type.R

```diff
@@ -0,0 +1,33 @@
+#' Lookup a mime type from a Google Drive type or file extension
+#'
+#' @param type Character. The Google Drive type or file extension you would
+#'   like to look up the mime type of.
```

Comment:
I think this should be the definitive place to document the `type` argument. And other functions should use `@inherit` or `@template`, so you don't repeat it everywhere.